src/main/java/org/opensrp/domain/IdentifierSource.java

```diff
@@ -0,0 +1,151 @@
+package org.opensrp.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.opensrp.util.IdentifierValidatorAlgorithm;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+
+public class IdentifierSource {
```

Comment:
can we  use lombok? check with Samuel

Reply:
Lombok is already configured and can be used

Reply:
Using getter/setter with lombok